pkg/grpc/device_definition.proto

```diff
@@ -17,62 +17,40 @@ message GetDeviceDefinitionItemResponse {
 	string device_definition_id = 1;
 	string name = 2;
 	string image_url = 3;
-	repeated  CompatibleIntegrations compatible_integrations = 4;
-	Type type = 5;
+	repeated  DeviceIntegration compatible_integrations = 4;
+	DeviceType type = 5;
 	VehicleInfo vehicle_data = 6;
 	bool  verified = 7;
-	repeated  DeviceIntegrations device_integrations = 8;
-	Make make = 9;
+	repeated  DeviceIntegration device_integrations = 8;
+	DeviceMake make = 9;
 	repeated DeviceStyles device_styles = 10;
 	string source = 11;
+}
 
-	message CompatibleIntegrations {
-		string id = 1;
-		string  type = 2;
-		string  style = 3;
-		string  vendor = 4;
-		string  region = 5;
-		string  country = 6;
-		string  capabilities = 7;
-	}
-
-	message Type {
-		string type = 1;
-		string  make = 2;
-		string  model = 3;
-		int32 year = 4;
-		repeated string sub_models = 5;
-	}
-
-	message DeviceIntegrations {
-		string id = 1;		
-		string type = 2;
-		string style = 3;
-		string vendor = 4;
-		string region = 5;
-		string country = 6;
-		string capabilities = 7;
-		string deviceDefinitionId = 8;
-	}
-
-	message DeviceStyles {
-		string id = 1;
-		string source = 2;
-		string name = 3;
-		string subModel = 4;
-		string deviceDefinitionId = 5;
-		string external_style_id = 6;
-	}
+message DeviceType {
+	string type = 1;
+	string  make = 2;
+	string  model = 3;
+	int32 year = 4;
+	repeated string sub_models = 5;
+}
 
-	message Make {
-		string id = 1;
-		string name = 2;
-		string logo_url = 3;
-		string oem_platform_name = 4;
-		uint64 token_id = 5;
-	}
+message DeviceStyles {
+	string id = 1;
+	string source = 2;
+	string name = 3;
+	string subModel = 4;
+	string deviceDefinitionId = 5;
+	string external_style_id = 6;
 }
 
+message DeviceMake {
+	string id = 1;
+	string name = 2;
+	string logo_url = 3;
+	string oem_platform_name = 4;
```

Comment:
What is this?

Reply:
`oem_platform_name` is like FordPass, OnStart, etc frontends wanted it at some point.